news/2020-11-29-a-labour-of-love.md

```diff
@@ -0,0 +1,104 @@
+
+We paired with *Kuba Oms* to set up a contest celebrating this timeless classic — a contest more ambitious than any we've hosted before!
+
+## Teams
+
+Lots to go through. Let's keep it short.
+
+**This is a team-based contest.** Users with different skillsets must collaborate to create a *full beatmap experience*, including:
+
+- Remix of the song **Kuba Oms - My Love** ([download stems here](https://assets.ppy.sh/contests/files/115/a_labour_of_love_stems.zip))
+- Beatmap spread following the [ranking criteria](/wiki/Ranking_Criteria) (osu! only)
+- Storyboard or custom video
+
+**Submit your team name and member list to [this form](https://docs.google.com/forms/d/e/1FAIpQLScjtVsUWIArvf--pKA0RLiCdXuO1_wO2Va2ICjenEcz9ZqI5Q/viewform).** Once confirmed, your team will be documented on the [A Labour of Love contest wiki page](/wiki/Contests/A_Labour_of_Love_Contest). Teams are limited to 5 members.
+
+With teams being public, we'll be taking a different approach from usual contests: **entries will NOT be anonymized.**
+
+In fact, we encourage you to tease your submission as it's being created! Feel free to announce your team publicly and show off your work. We're celebrating an osu! classic and we can't do that very well if everyone keeps their work hidden.
+
+*"I'm a super cool mapper/storyboarder, but I don't know anyone who can remix songs. What do I do?"* There's likely a lot of people in this situation. We recommend anyone looking for teammates to post in the comments saying so. Forming a suitable team is half of the challenge for this contest!
+
+## Schedule
+
+- **3 months:** Contest is open for submissions [here](https://osu.ppy.sh/community/contests/115) until the end of February 2021. There's a lot of work going into this, so you might need the extra time.
+- **2 weeks:** Submissions are judged by separate mapper and musician judging panels.
+- **2 weeks:** Community voting is opened for top 5 overall submissions.
+
+Once all of these are completed, the winners will be announced!
+
+## Prizes
+
+No such thing as a contest without prizes! Unlike most contests however, there's a few different ways to earn prizes this time around.
+
+Two groups will handle judging: **mappers (including storyboarders!)** and **musicians**. They'll be expected to evaluate the full package — the remixed song + beatmap + storyboard/video combo. 
+
+Each group has its own specialty of course, so we'll be rewarding prizes to the all members of the highest scoring team from each panel.
+
+- **Mapper panel**
+    - **Runner-up:** 3 months of osu!supporter 
+    - **Winner:**  6 months of osu!supporter + *Mappers' Pick* profile badge
+
+![Mappers' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_mapperspick.png)
+
+- **Musician panel**
+    - **Runner-up:** 3 months of osu!supporter 
+    - **Winner:**  6 months of osu!supporter + *Musicians' Pick* profile badge
```

Comment:
shouldnt the runner-up prize be below the winner?

Reply:
i think this layout works better since badge images are in the next line